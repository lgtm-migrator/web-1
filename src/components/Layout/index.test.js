import React from "react"
import renderer from "react-test-renderer"
import { Layout } from "."
import theme from "../../themes"

describe("<Layout /> component", () => {
  it("should render correctly with light theme", () => {
    const tree = renderer
      .create(<Layout theme={theme("light")}>Test children</Layout>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
