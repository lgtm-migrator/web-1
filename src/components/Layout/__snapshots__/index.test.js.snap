// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> component should render correctly with light theme 1`] = `
Array [
  <header
    className="MuiBox-root MuiBox-root-42 MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    itemScope={true}
    itemType="https://schema.org/WPHeader"
    role="banner"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h1
        className="MuiTypography-root MuiTypography-body1"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
          href="/"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_self"
        >
          
        </a>
      </h1>
    </div>
  </header>,
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <main
        className="MuiBox-root MuiBox-root-92 makeStyles-root-91"
        id="content"
        role="main"
        tabIndex={-1}
      >
        Test children
      </main>
    </div>
    <hr
      className="MuiDivider-root"
    />
    <footer
      className="MuiBox-root MuiBox-root-100"
      itemScope={true}
      itemType="https://schema.org/WPFooter"
      role="contentinfo"
    >
      <div
        className="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <ul
          className="MuiList-root makeStyles-root-101 MuiBox-root MuiBox-root-102 makeStyles-root-101"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <p
              className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
            >
              <small>
                © 
                2020
                 
                <strong />
              </small>
            </p>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/ajuda"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Ajuda
            </a>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/privacidade"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Privacidade
            </a>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/termos"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Termos de uso
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>,
]
`;

exports[`Default layout should render correctly with theme 1`] = `
Array [
  <header
    className="MuiBox-root MuiBox-root-42 MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    itemScope={true}
    itemType="https://schema.org/WPHeader"
    role="banner"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h1
        className="MuiTypography-root MuiTypography-body1"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
          href="/"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_self"
        >
          
        </a>
      </h1>
    </div>
  </header>,
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <main
        className="MuiBox-root MuiBox-root-92 makeStyles-root-91"
        id="content"
        role="main"
        tabIndex={-1}
      >
        Test children
      </main>
    </div>
    <hr
      className="MuiDivider-root"
    />
    <footer
      className="MuiBox-root MuiBox-root-100"
      itemScope={true}
      itemType="https://schema.org/WPFooter"
      role="contentinfo"
    >
      <div
        className="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <ul
          className="MuiList-root makeStyles-root-101 MuiBox-root MuiBox-root-102 makeStyles-root-101"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <p
              className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
            >
              <small>
                © 
                2020
                 
                <strong />
              </small>
            </p>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/ajuda"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Ajuda
            </a>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/privacidade"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Privacidade
            </a>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
              href="/termos"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              Termos de uso
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>,
]
`;
